lib/vagrant-sshfs/synced_folder.rb

```diff
@@ -126,6 +126,12 @@ def find_executable(name)
         ENV['PATH'] = oldpath
         return exepath
       end
+
+      # Enable rsync synced folders within WSL when in use
```

Comment:
```suggestion
      # Enable sshfs synced folders within WSL when in use
```

Reply:
this still needs to happen

Reply:
Thank you very much!
I have corrected that comment!